src/main/java/com/microsoft/graph/core/BaseClient.java

```diff
@@ -1,16 +1,16 @@
 // ------------------------------------------------------------------------------
 // Copyright (c) 2017 Microsoft Corporation
-// 
+//
```

Comment:
Maybe I forgot something but do we still need BaseClient, GraphServiceClioent and IBaseClient here? Aren't we supposed to use core with this?

Reply:
Same question for the files in the http namespace.

Reply:
This pr doesn't move anything, it's #615 that moves things into core. Until 615 we did have a lot of duplication and extra complexity.